reminders/apps.py

```diff
@@ -1,6 +1,19 @@
+from threading import Thread
 from django.apps import AppConfig
 
 
 class RemindersConfig(AppConfig):
     default_auto_field = 'django.db.models.BigAutoField'
     name = 'reminders'
+
+    scheduler = None
+
+    def ready(self) -> None:
+        if not self.scheduler:
+            Thread(target=RemindersConfig.create_scheduler, daemon=True).start()
```

Comment:
maybe use self instead like:
```suggestion
            Thread(target=self.create_scheduler(), daemon=True).start()
```

Reply:
done. thanks 😄

Reply:
please add the changes